spec/marc_util_spec.rb

```diff
@@ -0,0 +1,66 @@
+require 'spec_helper'
+
+describe 'MarcUtil' do
+  describe 'var_block' do
+    it 'should return nil if record is falsey' do
```

Comment:
Are these tests just placeholders for now?

Reply:
Yes, I started writing tests for the MarcUtil but then decided that the tests for the ItemStreamHandler were more important

Reply:
gotcha